pages/index.js

```diff
@@ -1,109 +1,181 @@
+              >
+                <div className="absolute bottom-0 w-full px-6 pb-5 pt-20 text-right text-white bg-gradient-to-t from-dark-600 to-transparent rounded-b-lg">
+                  <h3 className="text-white font-yrsa text-2xl">
+                    Rodina Cífkových
+                  </h3>
+                  <time
+                    className="text-sm text-accent-200"
+                    dateTime="2022-03-15T13:25:30+01:00"
+                  >
+                    15. 3. 2022
+                  </time>
+                  <p className="text-sm text-accent-200">
+                    Autor: Tereza Fatková
+                  </p>
+                </div>
+              </article>
+            </a>
+          </Link>
 
-    <p className="mx-8 md:mx-20 lg:mx-36">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In condimentum,
-      arcu vel tempus finibus, nulla metus congue lacus, ac vehicula ex nulla
-      non erat. Phasellus ultrices sapien sit amet eros ullamcorper lacinia. Sed
-      at sem facilisis ipsum fermentum pulvinar. Nam volutpat fringilla mi,
-      vitae mollis mi lobortis quis. Vivamus finibus ex est, interdum bibendum
-      dolor consequat in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
-    </p>
+          <Link href="/article">
+            <a className="font-medium tracking-wider text-xl flex-1">
+              <article
+                className='bg-[url("../styles/img/zamecek_vinice.jpg")] transition-opacity lg:hover:opacity-90 bg-stone-600 
+          bg-blend-overlay bg-cover bg-[position:50%_30%] bg-no-repeat rounded-md h-52 md:h-96 flex-1 relative'
+              >
+                <div className="absolute bottom-0 w-full px-6 pb-5 pt-20 text-right text-white bg-gradient-to-t from-dark-600 to-transparent rounded-b-lg">
+                  <h3 className="text-white font-yrsa text-2xl">
+                    Zámeček Vinice
+                  </h3>
 
-    <p className="mx-8 md:mx-20 lg:mx-36">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In condimentum,
-      arcu vel tempus finibus, nulla metus congue lacus, ac vehicula ex nulla
-      non erat. Phasellus ultrices sapien sit amet eros ullamcorper lacinia. Sed
-      at sem facilisis ipsum fermentum pulvinar. Nam volutpat fringilla mi,
-      vitae mollis mi lobortis quis. Vivamus finibus ex est, interdum bibendum
-      dolor consequat in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
-    </p>
-  </Layout>
-);
+                  <time
+                    className="text-sm text-accent-200"
+                    dateTime="2022-03-15T13:25:30+01:00"
+                  >
+                    15. 3. 2022
+                  </time>
+                  <p className="text-sm text-accent-200">
+                    Autor: Tereza Fatková
+                  </p>
+                </div>
+              </article>
+            </a>
+          </Link>
```

Comment:
Since this is repeated here and will be repeated even in the dynamic data. Consider abstracting it to an articleCard component.